skimage/segmentation/random_walker_segmentation.py

```diff
@@ -30,7 +30,7 @@ def new_del(self):
             pass
     umfpack.UmfpackContext.__del__ = new_del
     UmfpackContext = umfpack.UmfpackContext()
-except:
+except Exception:
```

Comment:
I didn't understand what could trigger this exception. Could we have a more specific `Exception` here? :blush:

Reply:
You are right, I specified the `ImportError` raised if umfpack is not installed ;-)